pipeline_dp/combiners.py

```diff
@@ -97,3 +99,29 @@ def merge_accumulators(self, count1: int, count2: int):
     def compute_metrics(self, count: int) -> float:
         return dp_computations.compute_dp_count(count,
                                                 self._params.mean_var_params)
+
+
+class MeanCombiner(Combiner):
+    """Combiner for computing DP Mean. Also returns sum and count in addition to
+    the mean.
+
+    The type of the accumulator is a tuple(count: int, sum: float) that holds
+    the count and sum of elements in the dataset for which this accumulator is
+    computed.
+    """
+
+    def __init__(self, params: CombinerParams):
+        self._params = params
+
+    def create_accumulator(self, values) -> (int, float):
+        return len(values), np.clip(values, self._params.aggregate_params.low,
+                                    self._params.aggregate_params.high).sum()
+
+    def merge_accumulators(self, accum1: tuple, accum2: tuple):
+        return accum1[0] + accum2[0], accum1[1] + accum2[1]
```

Comment:
Nit:
what do you think about the following for readbility:
count1, sum1 = accum1
count2, sum2 = accum2
and then use count, sum?
and the same for compute_metrics

Reply:
Makes sense, done.